### ISSUE TITLE
Fisherman's Heart

### ISSUE THREAD

**Issue by Deepupuu** [EdenServer/community#1230]
I know the chain of quests for Ebisu rod is under works with the pirate issue but Fisherman's Heart is separate and was wondering if maybe that quest is an easy fix.

It'd be nice to see stats on my fishing progress in the meantime, thanks!

**Comment by Setz0r** [comment]
yeah, individual fish catch recording is not currently happening.  other basic stats are though like general # of fish caught, lost, etc
